app/views/main.hbs

```diff
@@ -42,10 +46,23 @@
     {{! Do not track parameter }}
     <meta name="twitter:dnt" content="on">
 
-    {{! Web Monetization, if a payment pointer is provider (experimental) }}
+    {{! Web Monetization, if a payment pointer is provider (experimental)
+    TODO: figure out how to pass the content from .env or config }}
     {{#config.monetization}}
-    <meta name="monetization" content="{{.}}">
+      <meta name="monetization" content="$ilp.uphold.com/fnEfjW3apWmJ">
```

Comment:
I have this hardcoded atm because I haven't figured out how to properly pass it from .env yet

Reply:
This should already be happening -- I'll have to review with you to see why it's not working on your end.

Reply:
Before we merge this - can you also roll back this change please?